pkg/internal/codegen/parse/index.go

```diff
@@ -208,6 +209,41 @@ func parseResourceTag(tag string) resourceTags {
 	return result
 }
 
+// parseResourceAnnotation parses the tags in a "+resource=" comment into a resourceTags struct.
+func parseResourceAnnotation(t *types.Type) (resourceTags, error) {
+	result := resourceTags{}
+	for _, comment := range t.CommentLines {
+		anno := general.GetAnnotation(comment, "kubebuilder:resource")
+		if len(anno) == 0 {
+			continue
+		}
+		for _, elem := range strings.Split(anno, ",") {
+			key, value, err := general.ParseKV(elem)
+			if err != nil {
+				return resourceTags{}, fmt.Errorf("// +kubebuilder:resource: tags must be key value pairs.  Expected "+
+					"keys [path=<subresourcepath>] "+
```

Comment:
subresourcepath ?

Reply:
My bad. I will change that.:)

Reply:
that's not "subresourcepath", right?  It should just be "resourcepath", I think, because we're specifying resource aliases, not subresources like `Scale` (unless I'm misreading this).

Reply:
done!